js/main.js

```diff
@@ -1,4 +1,5 @@
 import './util.js';
+import './photo-rendering.js';
 import {photoDescriptions} from './data.js';
 
 photoDescriptions();
```

Comment:
По факту `util` ты тут не используешь, а используешь в другом блоке. 
С `photoDescriptions` то же самое: нет смысла создавать ещё одни моковые данные, которые никуда не идут.
```suggestion
import './photo-rendering.js';
```